src/components/CodeReminderModal/index.jsx

```diff
@@ -389,7 +379,11 @@ CodeReminderModal.propTypes = {
   initialValues: PropTypes.shape({}).isRequired,
   isBulkRemind: PropTypes.bool,
   selectedToggle: PropTypes.string,
-  data: PropTypes.shape({}),
+  data: PropTypes.shape({
+    selectedCodes: PropTypes.arrayOf(PropTypes.shape({})),
+    code: PropTypes.string,
+    email: PropTypes.string,
+  }),
```

Comment:
Nit: Could make this prop shape reusable for the other modal, but I don't think it's a huge deal here

Reply:
Good suggestion, but I think I'll hold off on that for now since we're likely going to be doing some refactoring of the modal components to make them more shareable between "Code Management" and "Subscription Management".